features/api/application_plans.feature

```diff
@@ -50,6 +50,11 @@ Feature: Application plans index page
     Then the plan is published
     And a buyer will be able to use it for their applications
 
+  Scenario: Unset the default application plan
+    When the default application plan is set
+    And an admin unsets the default application plan
+    Then the service will not have the default plan set
```

Comment:
Ideally this should also be tested (somehow):
```
    And any new application will use this plan
```

Reply:
"this plan" - which plan? :upside_down_face:  we have just unset it.

Yeah, I thought about verifying that the application will not be created if there is no default plan. But to be honest, I am not sure it belongs here... Not sure where it would belong though.

So, there are two different things related to the default application plans:

1. How to set and unset them: this PR focuses on the UI way, but there is also API way

2. How does the default application plan affects the behavior:
  - when a new buyer signs up or created, an application will be created only if there is a default application plan on the service - this applies for signing up both from the developer portal and the admin portal
  - when an application is created in the UI, the default plan is pre-filled in the creation form (if there is a default plan)
  
I guess you mean that we need to test the second point?

Well, maybe we need to check whether there are already tests for that, and if not add them. But I do not think it should be part of this PR. This PR just fixes the bug in the UI, and adds a test that verifies the fix.

We can add more checks in a different PR, if that's OK.

Reply:
Sorry big typo in my suggestion, what I mean is:
```
And any new application will NOT use this plan
```

Anyway, you're right, let's not make a big pile of cucumbers out of this PR. Checking both points 1 and 2 don't belong here.